README.md

```diff
@@ -123,6 +123,7 @@ The `vms` and `profile` variables can contain following attributes, note that if
 | serial_policy      | UNDEF                 | Specify a serial number policy for the Virtual Machine. Following options are supported. <br/><i>vm</i> - Sets the Virtual Machine's UUID as its serial number. <br/><i>host</i> - Sets the host's UUID as the Virtual Machine's serial number. <br/><i>custom</i> - Allows you to specify a custom serial number in serial_policy_value. |
 | serial_policy_value | UNDEF                 | Allows you to specify a custom serial number. This parameter is used only when <i>serial_policy</i> is custom. |
 | comment | UNDEF                             | Comment of the virtual Machine. |
+| merge_with_profile | UNDEF,False           | Permit to merge vm level cloud_init and sysprep properties with profile level |
```

Comment:
Would rename the variable. People who did not read the docs would misunderstand it.
How about `init_merge`? Opened for any other suggestion.
@jan-zmeskal @dangel101 any ideas?

Reply:
Instead of saying that it is undefined, but default False, please add the variable to the defaults/main.yml.
Also, formulate the description like https://github.com/oVirt/ovirt-ansible-vm-infra/pull/129/files#diff-04c6e90faac2675aa89e2176d2eec7d8R68

Reply:
thanks again for fast review 😃 

I set default value for merge_with_profile into defaults/main.yml, but i wait final decision to change name. 

I don't understand what means prefix `init_` ? Maybe for `cloud_init` ? but we also support merging of `sysprep` property 🤔

Reply:
WDYT about init_merge for cloud_init and sysprep_merge for sysprep ?